### PR TITLE
Updating docs to address #8

### PR DESCRIPTION
**Pull request by frederikgeth**
@ccoffrin, is this how you see #8? If so, don't hesitate to merge!

I copied the bibtex strings straight from IEEE explore where possible.

### CODE REVIEW

File: src/form/acp.jl [hunk 1]
```diff
@@ -9,7 +9,19 @@ abstract type AbstractACPForm <: AbstractPowerFormulation end
...
+```
+@techreport{Cain2012,
+author = {Cain, Mary B and {O' Neill}, Richard P and Castillo, Anya},
+pages = {1--36},
+title = {{History of optimal power flow and formulations}},
+url = {https://www.ferc.gov/industries/electric/indus-act/market-planning/opf-papers/acopf-1-history-formulation-testing.pdf}
+year = {2012}
+}
+```
+"""
```
Comment (ccoffrin) [inline src/form/acp.jl]: Maybe we should also add,
```
@article{carpentier1962contribution,
  title={Contribution to the economic dispatch problem},
  author={Carpentier, J},
  journal={Bulletin de la Societe Francoise des Electriciens},
  volume={3},
  number={8},
  pages={431--447},
  year={1962}
}
```
it is the earliest refrence I know of.

Comment (frederikgeth) [reply src/form/acp.jl]: added

**Comment by frederikgeth** [comment]
done, keywords have been dropped

**Comment by ccoffrin** [comment]
Looks good to me.  Thanks!

**Pull request merged**